### PR TITLE
Add "registerPermissions"

### PR DESCRIPTION
**Pull request by webmaxx**
Plugin uses 'rainlab.notify.manage_notifications' in controller 'Notifications', but it permission is not registered.

### CODE REVIEW

**Comment by LukeTowers** [comment]
@webmaxx please translate the permission string, also I don't think that SettingsManager constant exists, does it?

**Comment by webmaxx** [comment]
Constant [System\Classes\SettingsManager::CATEGORY_NOTIFICATIONS](https://github.com/octobercms/october/blob/2dbc56aead2ef2355135349b83860457f48f6235/modules/system/classes/SettingsManager.php#L34), uses in [registerSettings](https://github.com/rainlab/notify-plugin/blob/49db4ec724630bb0205f5360e3121e89826b9043/Plugin.php#L28) method and contains translate string 'system::lang.system.categories.notifications'.

File: Plugin.php [hunk 1]
```diff
@@ -48,4 +48,14 @@ public function registerNotificationRules()
             ],
         ];
     }
+
+    public function registerPermissions()
+    {
+        return [
+            'rainlab.notify.manage_notifications' => [
+                'tab' => SettingsManager::CATEGORY_NOTIFICATIONS,
+                'label' => 'Notifications management'
```
Comment (LukeTowers) [inline Plugin.php]: This is the permission string I meant that needs to be translated.

Comment (webmaxx) [reply Plugin.php]: Sorry. I did not immediately understand.

**Pull request merged**